fog/view/server/src/router_admin_service.rs

```diff
@@ -62,7 +66,11 @@ impl FogViewRouterAdminService {
             ChannelBuilder::default_channel_builder(grpc_env)
                 .connect_to_uri(&view_store_uri, logger),
         );
-        shard_clients.insert(view_store_uri, Arc::new(view_store_client));
+        // TODO: Add block range or sharding strategy to this...
+        // Check to make sure this block range isn't already covered...
```

Comment:
I actually think overlapping block ranges is fine, desirable even.

Reply:
Yes that's true. This comment means that there shouldn't be any duplicate block ranges (which is taken care of by the above logic, so I removed this comment).